src/shared/ui/logo/logo.module.scss

```diff
@@ -1,11 +1,12 @@
 .logo {
   @include media-tablet {
-    max-width: 40px;
-    max-height: 40px;
+    width: 40px;
+    height: 40px;
   }
 
-  max-width: 56px;
-  max-height: 56px;
+  width: 56px;
+  max-width: 100%;
```

Comment:
Could you please explain why we need max-width: 100% here?

Reply:
without max-width it works same, i think you can delete it safely